unicodetools/data/ucd/dev/VerticalOrientation.txt

```diff
@@ -2178,6 +2178,7 @@ FFFC..FFFD     ; U  # So     [2] OBJECT REPLACEMENT CHARACTER..REPLACEMENT CHARA
 1BC9D..1BC9E   ; R  # Mn     [2] DUPLOYAN THICK LETTER SELECTOR..DUPLOYAN DOUBLE MARK
 1BC9F          ; R  # Po         DUPLOYAN PUNCTUATION CHINOOK FULL STOP
 1BCA0..1BCA3   ; R  # Cf     [4] SHORTHAND FORMAT LETTER OVERLAP..SHORTHAND FORMAT UP STEP
+1CEB0..1CEB3   ; R  # So     [4] HORIZONTAL ZIGZAG LINE..BLACK RIGHT TRIANGLE CARET
```

Comment:
Since this file says code points “that are not listed[…] are given the value R”, we may want to _not_ add new characters since that feels like we are saying we have some evidence of how they ought to be used in vertical typesetting when we don’t.

Reply:
Looking at the current version of the file, it looks like it lists every character that is assigned or have a value different from R. That is, it seems to omit only those R code points that are unassigned. And, this file gets regenerated, so omitting certain characters would require both a policy change and a code change.

We should remain consistent with current practice until we (PAG+UTC) decide otherwise.

Reply:
While this is unwritten in the file, all assigned code points are automatically listed, even when we don’t have a clue about their vertical typesetting. I don’t think we want to change that—if we want to have information in here we want it to part of the property. Maybe we should have a new value No_Clue, but that seems like another can of worms.